.circleci/config.yml

```diff
@@ -22,6 +22,7 @@ jobs:
       - run: go get github.com/mattn/goveralls
       - run: curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
       - run: make build
+      - run: make test-all
```

Comment:
this will end up running the integration tests twice with the line below. could just switch this one to `make test-unit`

Reply:
The idea is to call all test, if I switch it for test-unit then I understand that the integration test will not be executed. Or, are you saying that was created a command that calls *unit* tests and it calls the integration tests as well?

Reply:
I catcher the test-integration-cover is calling the integration and its coveralls